src/modules/auto-google-map.js

```diff
@@ -0,0 +1,172 @@
+/**
+ * Auto Google Map module
+ * @author Deux Huit Huit
+ * @requires com/google-map.js
+ */
+(function ($, undefined) {
+
+	'use strict';
+
+	var scope = $('.page');
+
+	var sels = {
+		ctn: '.js-auto-google-map-ctn',
+		map: '.js-auto-google-map',
+		marker: '.js-auto-google-map-marker',
+		styles: '.js-auto-google-map-style'
+	};
+
+	var ready = false;
+
+	var DEFAULT_MARKER_WIDTH = 30;
+	var DEFAULT_MARKER_HEIGHT = 30;
+	var DEFAULT_ZOOM = 10;
+
+	var createMarker = function (element, mapComponent) {
+		var lat = element.attr('data-lat');
+		var lng = element.attr('data-lng');
+		var iconSrc = element.attr('data-icon');
+		var iconWidth = parseInt(element.attr('data-icon-width'), 10) || DEFAULT_MARKER_WIDTH;
+		var iconHeight = parseInt(element.attr('data-icon-height'), 10) || DEFAULT_MARKER_HEIGHT;
+
+		if (!!App.device.internetexplorer) {
+			iconSrc = null;
+		}
+
+		var config = {
+			id: element.attr('data-poi-id'),
+			LatLng: new window.google.maps.LatLng(lat, lng),
+			content: element.html(),
+			iconImage: {
+				src: iconSrc,
+				size: new google.maps.Size(iconWidth, iconHeight),
+				scaledSize: new google.maps.Size(iconWidth, iconHeight),
+				p1: {
+					x: 0,
+					y: 0
+				},
+				p2: {
+					x: iconWidth / 2,
+					y: iconHeight
+				}
+			},
+			zIndex: 50
+		};
+
+		if (!iconSrc) {
+			delete config.iconImage;
```

Comment:
Set the property to undefined, then assign something do it instead

Reply:
@fhamon ping!

Reply:
out daaaaaaated